src/components/userContent/cardTemplate/card.js

```diff
@@ -0,0 +1,63 @@
+import './card.css';
+import card from './card.hbs';
+import Header from './cardHeader/header';
+import OrderItem from './orderItem/orderItem';
+import ShowMore from './showMore/showMore';
+
+
+export default class Card {
+  render(target, props) {
+    let orderPrice = 0;
+    for (const order of props.orders) {
+      orderPrice += order.price;
+    }
+
+    const cardContext = {
+      orderPrice,
+      active: props.header.active,
+      button: props.orders.length > 0 ? 'Редактировать' : 'Заказать',
+      sumPrice: props.orders.length > 0,
+    };
+
+    const cardTemplate = card(cardContext);
+    target.innerHTML = cardTemplate;
+
+    if (props.header.active) {
+      props.header.headerStyle = 'active-card';
```

Comment:
Mutating props object is discouraged. If you need to modify some attribute of an object create a new object.
Example:

```
const header = new Header();
header.render(target.querySelector('.header'), createHeaderProps(props.header));

...

createHeaderProps(headerProps) {
    return {
       ...headerProps,
       headerStyle: props.header.active ? 'active-card' : 'inactive-card'
    }
}
```